resources/less/forum.less

```diff
@@ -60,10 +60,16 @@
   }
 }
 
-@media only screen and (max-device-width: 480px) {
+@media only screen and (max-device-width: 560px) {
```

Comment:
Could we use the variables from core? I think those should be available.

Reply:
It looks best on 580px width, but if other extensions add their items there, there may be a problem with overlapping again.
So it's a good idea with these core variables. I will rebuild these styles a bit.

Reply:
I added some changes. Now the div with the best answer goes to the second line only if it doesn't fit in the first line